ci/badge/.htaccess

```diff
@@ -0,0 +1,22 @@
+#Prevent the image caching in Github                                                                               
+<IfModule mod_headers.c>
+    Header set Cache-Control "no-cache, no-store, must-revalidate"
+    Header set Pragma "no-cache"
+    Header set Expires 0
+</IfModule>
+ 
+<FilesMatch "\.(css|flv|gif|htm|html|ico|jpe|jpeg|jpg|js|mp3|mp4|png|pdf|swf|txt)$">
+    <IfModule mod_expires.c>
+        ExpiresActive Off 
+    </IfModule>
+    <IfModule mod_headers.c>
+        FileETag None
+        Header unset ETag
+        Header unset Pragma
+        Header unset Cache-Control
+        Header unset Last-Modified
+        Header set Pragma "no-cache"
+        Header set Cache-Control "max-age=0, no-cache, no-store, must-revalidate"
+        Header set Expires "Thu, 1 Jan 1970 00:00:00 GMT"
```

Comment:
One question, Do we have to set-up the past date (e.g., 1970) to enable the  "**no-cache**" approach?

Reply:
These days, the expire header looks unnecessary.
reference link: https://stackoverflow.com/questions/49547/how-do-we-control-web-page-caching-across-all-browsers
`Expires with a date in the past for HTTP/1.0 clients (although real HTTP/1.0-only clients are completely non-existent these days).`

Reply:
@gichan-jang , Please remove unnecessary statements to do code maintenance efficiently in the future.

Reply:
Removed!